src/ai/.x/templates/openai-asst-webpage-ts/_.json

```diff
@@ -0,0 +1,14 @@
+{
+  "_LongName": "OpenAI Assistants Webpage",
+  "_ShortName": "openai-asst-webpage",
+  "_Language": "TypeScript",
+  "ClassName": "OpenAIAssistantsStreamingClass",
+  "AZURE_OPENAI_ASSISTANT_ID": "<insert your OpenAI assistant ID here>",
+  "AZURE_OPENAI_API_KEY": "<insert your Azure OpenAI API key here>",
+  "AZURE_OPENAI_API_VERSION": "<insert your Azure OpenAI API version here>",
+  "AZURE_OPENAI_ENDPOINT": "<insert your Azure OpenAI endpoint here>",
+  "AZURE_OPENAI_CHAT_DEPLOYMENT": "<insert your Azure OpenAI chat deployment name here>",
+  "OPENAI_API_KEY": "<insert your OpenAI API key here>",
+  "OPENAI_ORG_ID": "<insert your OpenAI organization ID here>",
+  "OPENAI_MODEL_NAME": "<insert your OpenAI model name here>"
```

Comment:
I see there are other missing env vars:
- AZURE_OPENAI_CHAT_MODEL
- AZURE_OPENAI_EMBEDDING_DEPLOYMENT
- AZURE_OPENAI_EMBEDDING_MODEL
- AZURE_OPENAI_EVALUATION_DEPLOYMENT
- AZURE_OPENAI_EVALUATION_MODEL
- AZURE_OPENAI_KEY

Reply:
i updated it to be symmetric with the other ones, and i removed the embedding and evalution vars as they're not needed in this level of the sample.